components/kyma-environment-broker/internal/cis/account_cleanup.go

```diff
@@ -77,7 +77,7 @@ func (ac *SubAccountCleanupService) executeDeprovisioning(subaccounts []string,
 	}
 
 	for _, instance := range instances {
-		operation, err := ac.brokerClient.Deprovision(instance)
+		operation, err := ac.brokerClient.Deprovision(instance, true)
```

Comment:
> 2) Adjust subaccount deletion job to send such a kymaContext to KEB

@PK85 is this the subaccount cleanup you mentioned in https://github.com/kyma-project/control-plane/issues/2512#issue-1596917634 ?